KUBERNETES.md

```diff
@@ -0,0 +1,241 @@
+
+```bash
+kubectl --namespace kube-system get deploy/tiller-deploy
+```
+
+```
+NAME            DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+tiller-deploy   1         1         1            1           3m
+```
+
+Install the sandbox. Since Chirper only uses Cassandra, we're disabling the other services but you can leave them enabled by omitting the `set` flag if you wish.
+
+```bash
+helm install lightbend-helm-charts/reactive-sandbox --name reactive-sandbox --set elasticsearch.enabled=false,kafka.enabled=false,zookeeper.enabled=false
+```
+
+Verify that it is available (this takes a minute or two):
+
+```bash
+kubectl get deploy/reactive-sandbox
+```
+
+```
+NAME               DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+reactive-sandbox   1         1         1            1           1m
+```
+
+##### Build Project
+
+```bash
+sbt clean docker:publishLocal
+```
+
+##### View Images
+
+```bash
+docker images
+```
+
+##### Deploy Projects
+
+Finally, you're ready to deploy the services. Be sure to adjust the secret variables and cassanda service address as necessary.
+
+```bash
+# Be sure to change these secret values
+
+chirp_secret="youmustchangeme"
+friend_secret="youmustchangeme"
+activity_stream_secret="youmustchangeme"
+front_end_secret="youmustchangeme"
+
+# Default address for reactive-sandbox, change if using external Cassandra
+
+cassandra_svc="_cql._tcp.reactive-sandbox-cassandra.default.svc.cluster.local"
+
+# Configure the services to allow requests to the minikube IP (Play's Allowed Hosts Filter)
+
+allowed_host="$(minikube ip)"
+
+# deploy chirp-impl
+
+rp generate-kubernetes-resources "chirp-impl:1.0.0-SNAPSHOT" \
+  --generate-pod-controllers --generate-services \
+  --env JAVA_OPTS="-Dplay.http.secret.key=$chirp_secret -Dplay.filters.hosts.allowed.0=$allowed_host" \
+  --external-service "cas_native=$cassandra_svc" \
+  --service-type NodePort \
+  --pod-controller-replicas 2 | kubectl apply -f -
+
+# deploy friend-impl
+
+rp generate-kubernetes-resources "friend-impl:1.0.0-SNAPSHOT" \
+  --generate-pod-controllers --generate-services \
+  --env JAVA_OPTS="-Dplay.http.secret.key=$friend_secret -Dplay.filters.hosts.allowed.0=$allowed_host" \
+  --external-service "cas_native=$cassandra_svc" \
+  --pod-controller-replicas 2 | kubectl apply -f -
+
+# deploy activity-stream-impl
+
+rp generate-kubernetes-resources "activity-stream-impl:1.0.0-SNAPSHOT" \
+  --generate-pod-controllers --generate-services \
+  --env JAVA_OPTS="-Dplay.http.secret.key=$activity_stream_secret -Dplay.filters.hosts.allowed.0=$allowed_host" | kubectl apply -f -
+
+# deploy front-end
+
+rp generate-kubernetes-resources "front-end:1.0.0-SNAPSHOT" \
+  --generate-pod-controllers --generate-services \
+  --env JAVA_OPTS="-Dplay.http.secret.key=$front_end_secret -Dplay.filters.hosts.allowed.0=$allowed_host" | kubectl apply -f -
+
+# deploy ingress
+rp generate-kubernetes-resources \
+  --ingress-path-suffix '*' \
```

Comment:
This line can be removed - it was from a test on one of the cloud providers. I think our commands here should focus on Minikube.